### ISSUE TITLE
ft.search support regex

### ISSUE THREAD

**Issue by chenfengone** [RediSearch/RediSearch#1922]
Seems it can't search with regular expression, like other db (mysql) did:
SELECT name FROM person_tbl WHERE name REGEXP '^st';

don't know do you have any plan to support this feature?

**Comment by emmanuelkeller** [comment]
@chenfengone Yes, the support of regular expression is part of the roadmap. It should be available later this year.

**Comment by chenfengone** [comment]
Thanks @emmanuelkeller , looking forward this feature

**Comment by anderslangballe** [comment]
@emmanuelkeller has this been implemented yet? I would also be very interested in this feature.
